object/splitinfo.go

```diff
@@ -56,13 +62,15 @@ func (s SplitInfo) LastPart() (v oid.ID, isSet bool) {
 	return
 }
 
+// SetLastPart sets last object id.
 func (s *SplitInfo) SetLastPart(v oid.ID) {
 	var idV2 refs.ObjectID
 	v.WriteToV2(&idV2)
 
 	(*object.SplitInfo)(s).SetLastPart(&idV2)
 }
 
+// Link returns linker object id and a flag was it set or not.
```

Comment:
```suggestion
// Link returns a linker object.
// Second returning value is a flag, indicating if the last part is set or not.
```

just cause u already use such wording above

Reply:
BTW i have never read "linker object" before, we usually write "link object". even the getter is called `Link`

@cthulhu-rider, @roman-khimov